src/core/connectivity.jl

```diff
@@ -253,6 +265,18 @@ struct LatticeConnectivity <: AbstractConnectivity
     end
 end
 
+function Base.isequal(c0::LatticeConnectivity, c1::LatticeConnectivity)::Bool
+    if c0.dimensions != c1.dimensions
+        return false
+    end
+
+    if c0.excluded_connections != c1.excluded_connections
+        return false
+    end
+
+    return c0.excluded_positions == c1.excluded_positions
```

Comment:
The `excluded_positions` aren't sorted for the `LatticeConnectivity`.

Reply:
refactored the isequal for a new type ExcludedPositions